### PR TITLE
Updated the two managers (gait and motion)

### PR DESCRIPTION
**Pull request by DavidMansolino**
- Use of the new class to initiate the LinuxMotionTimer.
- Replaced loop for Joint enabling by SetEnableBodyWithoutHead in the gait manager.
- Added a new attribute to the class DARwinOPGaitManager to save the walking instance.

### CODE REVIEW

File: resources/projects/robots/darwin-op/include/managers/DARwInOPGaitManager.hpp [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 namespace Robot {
   class Action;
```
Comment (DavidMansolino) [inline resources/projects/robots/darwin-op/include/managers/DARwInOPGaitManager.hpp]: Is this line realy usefull ?

Comment (fabienrohrer) [inline resources/projects/robots/darwin-op/include/managers/DARwInOPGaitManager.hpp]: Effectively, it seems not.

Comment (fabienrohrer) [inline resources/projects/robots/darwin-op/include/managers/DARwInOPGaitManager.hpp]: Could you fix it (try to compile to be sure) and modify this PR?

Comment (DavidMansolino) [inline resources/projects/robots/darwin-op/include/managers/DARwInOPGaitManager.hpp]: indeed, it seems to work perfectly without it.
I have fixed it.

**Pull request merged**